services/secondary-storage/controller/src/main/java/org/apache/cloudstack/secondarystorage/SecondaryStorageManagerImpl.java

```diff
@@ -1272,22 +1216,20 @@ public boolean finalizeCommandsOnStart(Commands cmds, VirtualMachineProfile prof
     public boolean finalizeStart(VirtualMachineProfile profile, long hostId, Commands cmds, ReservationContext context) {
         CheckSshAnswer answer = (CheckSshAnswer)cmds.getAnswer("checkSsh");
         if (!answer.getResult()) {
-            s_logger.warn("Unable to ssh to the VM: " + answer.getDetails());
+            s_logger.warn(String.format("Unable to connect via SSH to the VM [%s] due to [%s] ", profile.toString(), answer.getDetails()));
             return false;
         }
 
         try {
-            //get system ip and create static nat rule for the vm in case of basic networking with EIP/ELB
```

Comment:
Observed that all the comments are removed. I think these comments were added to understand the code there, and would help in debugging sometimes. You can remove them judiciously.

Reply:
Comments were removed because they are nonsense. Most of they only describe what the next line is doing, e.q.: 
```java
//verify if var is true then print it in console
if (var) {
	System.out.println("var is true");
}
```

These types of comments do not improve debugging and goes against a clean code. However, there are comments that explain a bit of the process, but they could be resolved with an extraction of the code to a method with a corresponding name, javadoc and etc.

Also, there are `TODO` comments. These requests should be regularized in issues, not in code.

Comments should be a last resort, not the rule. If we take all the comments and sort then in a logic way, we could write a book :smile: .

If there are some points you really think that should be kept, let me know, and then we can discuss if it is important to maintain it; or, maybe, we are able to get ride of it by executing some minor refactoring.

Reply:
I agree;

1. comment last resort
2. (replace by) method name first resort

in this case I only wonder about the `EIP/ELB` part of the comment, so for me it was only adding confusion.

Reply:
> Comments were removed because they are nonsense. Most of they only describe what the next line is doing, e.q.:
> 
> ```java
> //verify if var is true then print it in console
> if (var) {
> 	System.out.println("var is true");
> }
> ```
> 
> These types of comments do not improve debugging and goes against a clean code. However, there are comments that explain a bit of the process, but they could be resolved with an extraction of the code to a method with a corresponding name, javadoc and etc.
> 
> Also, there are `TODO` comments. These requests should be regularized in issues, not in code.
> 
> Comments should be a last resort, not the rule. If we take all the comments and sort then in a logic way, we could write a book 😄 .
> 
> If there are some points you really think that should be kept, let me know, and then we can discuss if it is important to maintain it; or, maybe, we are able to get ride of it by executing some minor refactoring.

@GutoVeronezi you can remove such nonsense comments, even i'm not in favour of such comments. i cannot check and advise on every comment. you can take a judicious call while refactoring them. thanks.